internal/internal_nexus_task_handler.go

```diff
@@ -444,9 +467,17 @@ func (h *nexusTaskHandler) nexusHandlerErrorToProto(handlerErr *nexus.HandlerErr
 	if err != nil {
```

Comment:
Nit: it seems like we have all the Nexus error conversion logic here https://github.com/temporalio/sdk-go/pull/1802/files#diff-82a19cd3836c5a75e55d9c5d4400c0bfdd09029eb29fd1fa9755cf1a724f4654R86 except this one function. Not blocking but might be nice to group it.

Reply:
There's maybe a slight duplication but I don't think it's very critical apart from the grouping.